db/migrate/20181025225740_remove_nutshell_columns.rb

```diff
@@ -0,0 +1,13 @@
+class RemoveNutshellColumns < ActiveRecord::Migration
+  def up
+    remove_index :users, :nutshell_id
+    remove_column :users, :nutshell_id
+    remove_column :affiliates, :nutshell_id
+  end
+
+  def down
+    add_column :affiliates, :nutshell_id, :integer
+    add_column :users, :nutshell_id, :integer
+    add_index :users, :nutshell_id
+  end
+end
```

Comment:
The change method works for removing_index and remove_column.  
https://edgeguides.rubyonrails.org/active_record_migrations.html#using-the-change-method

Reply:
Migrations do not get run automatically correct?  This should not be run till after the code is pushed.

Reply:
Ah - my notes on when `remove_column` could be used were from an older version of Rails. Thanks, will update.